grammar.js

```diff
@@ -146,35 +146,23 @@ const rules = {
       '<<<',
       $._heredoc_start,
       opt(alias($._heredoc_start_newline, '\n')),
-      choice.rep($._heredoc_body, $.variable, $.embedded_brace_expression),
+      choice.rep($._heredoc_body, $.variable, $.embedded_braced_expression),
```

Comment:
@frankeld do you know how much work if any there is on the Semgrep side if we change node names for existing rules? Trying to determine if this name change is worth the hassle.

Reply:
Thanks for asking - very easy fix on the Semgrep side. Happy to have this terminology normalization.